cinema/views.py

```diff
@@ -196,8 +196,9 @@ def get_serializer_class(self):
                 "date",
                 type=OpenApiTypes.DATE,
                 description=(
-                    "Filter by datetime of MovieSession "
-                    "(ex. ?date=2022-10-23)"
+                    "Filter by datetime of MovieSession " "("
+                    "ex. ?date=2022-10-23"
+                    ")"
```

Comment:
The string concatenation for the description in the `OpenApiParameter` for the `date` parameter could be simplified by using a single string with parentheses. This would improve readability and maintainability.